website/docs/docs/ai/quick-start.md

```diff
@@ -0,0 +1,208 @@
+# Quick Start
+
+
+## Enabling the feature
+
+
+This feature is disabled by default on all organizations. To enable the feature:
+
+1. In the UI, navigate to "Settings" -> "Preferences" -> "System Preferences".
+2. Check the "AI Test Generation" checkbox.
+3. Save the settings.
+
+:::note
+You must be an Administrator or a user with the [`system_preference:manage:*` permission](https://docs.pactflow.io/docs/permissions#system_preferencemanage) to perform this action.
+:::
+
+## Installation
+
+For *nix users (including Windows users running WSL/msys2/mingw), use the following command to download and install:
+
+```sh
+curl https://download.pactflow.io/ai/get.sh | sh
+```
+
+For Windows PowerShell users, use the following command to download and install:
+
+```pwsh
+Invoke-WebRequest -Uri https://download.pactflow.io/ai/get.ps1 | Invoke-Expression
+```
+
+### Installation Options
+
+There are some options which you can set during installation.
+
+For a full list, see the `--help`/`-h` command
+
+```sh
+curl https://download.pactflow.io/ai/get.sh | sh -s -- -h
+```
+
+- `--verbose` / `-v` / `PACTFLOW_AI_VERBOSE`: Enable verbose output.
+- `--quiet` / `-q` / `PACTFLOW_AI_QUIET`: Disable progress output.
+- `--yes` / `-y` / `PACTFLOW_AI_YES`: Disable confirmation prompt and assume 'yes'.
+- `--destination` / `-d` / `PACTFLOW_AI_DESTINATION`: Specify the directory to install the binary.
+- `--default-host` / `PACTFLOW_AI_DEFAULT_HOST`: Choose a default host triple rather than autodetecting.
+- `--no-modify-path` / `PACTFLOW_AI_NO_MODIFY_PATH`: Don't configure the PATH environment variable.
+
+Verify the installation by running `pactflow-ai` to ensure it executes successfully.
+
+### Manual installation
+
+Alternatively, download the latest version for your OS and architecture from the table below. Be sure to add it to your environment's `PATH`:
+
+<AiDownloadTable />
```

Comment:
This page is missing the import for the table, so it does not render


`import { AiDownloadTable } from "../../../src/components/index";`

Reply:
good spot - thanks! (and fixed)